controllers/facebook_controller.go

```
@@ -123,6 +100,12 @@ func (controller *FacebookController) tokenHandler(w http.ResponseWriter, r *htt
 			errorlist.GenericDataAccessError)
 	} else if !errors.IsEmpty() {
 		setRequestErrors(resp, errors, "unable to lookup user from access token")
+	} else if controller.permissionGateway.LoginMethodBlocked(
```

Hm I think maybe it would make more sense to just rename the current gateway to `AuthorizationService` in that case, since why have an abstraction on top of another abstraction if we can avoid it?

Also I think the method should stay named `LoginMethodBlocked`. I'm specifically ignoring any errors that come up during the http request to the permission service and just returning `false` in that case, since 99% of logins won't be blocked anyway. So I think that it's a good idea to have 

```
func (s *authorizationService) LoginMethodBlocked(userId, method string) bool {
  blocked, err :=  s.authorized(userId, method)
  if err != nil {
    // log
    return false
  }

  return blocked
}

func (s *authorizationService) authorize(userId, action string) (bool, error) {
  // ask service
}
```